### DISCUSSION TITLE
Are we allowed to add additional columns?

### DISCUSSION THREAD

**Discussion by micburke99** [CMSgov/hospital-price-transparency#51] [Q&A] [answered]
Are we allowed to add additional columns for different payers or should each payer be listed on a separate row? Same question for differentiating our Inpatient vs Outpatient rates?  Should an additional column be added on the same procedure line, or should a new row be created for the same procedure per procedure, per rate?

**Comment by RidenourK** [comment, marked as answer]
@micburke99. Thank you for your question. 

As of July 1, 2024, hospitals’ machine-readable files must conform to a CMS template layout, data specifications, and data dictionary. CMS has made the CMS template available in three non-proprietary formats, CSV “tall”, CSV “wide”, and JSON, in the “[Documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation)” section of this CMS Hospital Price Transparency - Data Dictionary GitHub repository.

As indicated in the CY 2024 OPPS/ASC Final Rule, the CSV “tall’’ format contains separate payer and plan data elements that provide the benefit of static header naming with less opportunity for standardization error and that is similar to existing output files that many hospitals have been using to build their MRFs. The CSV ‘‘wide’’ format contains variable payer-specific negotiated charge data elements that incorporate the payer and plan name into a single column header, which may reduce the file size because many data elements would not need to be repeated as frequently. Please refer to the discussion at [88 FR 82088](https://www.federalregister.gov/d/2023-24293/p-4893).

CMS strongly recommends hospitals start by downloading one of the template layouts or schema to create the machine-readable files. Please then review the technical instructions in the “[Documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation)” section, followed by the “[Examples](https://github.com/CMSgov/hospital-price-transparency/tree/master/examples)” available on this the CMS Hospital Price Transparency - Data Dictionary GitHub repository for more information about how to encode data in one of the CMS templates.
